module/PowerNSX.psm1

```diff
@@ -10697,11 +10697,11 @@ function New-NsxLogicalRouterInterfaceSpec {
 
     .EXAMPLE
 
-    PS C:\> $Uplink = New-NsxLogicalRouterinterfaceSpec -Name Uplink_interface -Type
+    $Uplink = New-NsxLogicalRouterinterfaceSpec -Name Uplink_interface -Type
```

Comment:
It is multiple command example, it is better to kept PS C:\> ?

(and also example look strange because not on the same line

Reply:
Per formatting template of PoSH. 
Remove PS /> or PS C:\> 
Line break at 80 characters.

Example sweeps are happening. Not all examples have been touched yet.

Reply:
for multiline examples where there is code, the first line immediately after the .EXAMPLE should _not_ have PS C:\> (get-help automatically adds this), but subsequent code lines should have it manually, so it displays consistently in get-help, with a prompt for all code lines. (but not comments...)